lib/screens/v2/widget_instance/subway_status.ex

```diff
@@ -421,9 +444,11 @@ defmodule Screens.V2.WidgetInstance.SubwayStatus do
   end
 
   def serialize_green_line_branch_alert(
```

Comment:
This seems like a good opportunity to add a typespec to this public function.

Reply:
Hmm, this function shouldn't be public actually. Will switch to private. I still think this and `serialize_alert/2` could benefit from typespecs so will add them.